src/containers/profile/EditProfile.js

```diff
@@ -4,25 +4,25 @@ import { Button, Icon } from 'antd';
 import { bindActionCreators } from 'redux';
 import EditProfileForm from '../../components/profile/forms/EditProfileForm';
 
-import { editUserProfile } from '../../actions/profile/profile';
+import { editUserProfile } from '../../actions/profileActions';
 import { displayModalActions, hideModalActions } from '../../actions/modalActions';
 
 export class UnconnectedEditProfile extends React.Component {
   showModal = () => {
-    const { displayModalActions } = this.props;
+    const { displayModalActions } = this.props; // eslint-disable-line
     displayModalActions();
   };
 
   handleCancel = () => {
-    const { hideModalActions } = this.props;
+    const { hideModalActions } = this.props; // eslint-disable-line
     hideModalActions();
   };
 
```

Comment:
Function in line 33 can be written in one line:

saveFormRef = formRef => this.formRef = formRef;

Reply:
Also, destructuring in the render can be in one line